src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,41 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    font-family: $main-font-family;
+    @include text($color-dark-blue, 12px, 500);
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 23px 0;
+    position: relative;
+  }
+
+  &__link:hover {
+    color: $color-cyan;
+  }
+}
+
+.is-active {
```

Comment:
let's create separate class as modifier for the link (nav__link--is-active), and move styles to this class

Reply:
I was going to do with modifier at first but there was a condition in the task "add class `is-active` to the first link (`Apple`) in navigation". When I remove is-active class and use modifier not all tests pass. The error "No class is-active" occures. So I added also modifier and is-active class.